webportal/webportal/templates/public/calculate.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>my login</title>
+</head>
+<body>
```

Comment:
Remove these lines, 
look in another HTml pages ,

refer this : https://github.com/joynewton1996/Webportal/blob/master/webportal/webportal/templates/public/about.html

use like :
``` 
{% extends "layout.html" %}

{% block content %}

<.YOUR HTML FORM CODE >

{% endblock %}

```